src/OVAL/oval_generator.c

```diff
@@ -152,8 +191,23 @@ xmlNode *oval_generator_to_dom(struct oval_generator *generator, xmlDocPtr doc,
 		xmlNewTextChild(gen_node, ns_common, BAD_CAST "product_name", BAD_CAST generator->product_name);
 	if (generator->product_version)
 		xmlNewTextChild(gen_node, ns_common, BAD_CAST "product_version", BAD_CAST generator->product_version);
-	if (generator->schema_version)
-		xmlNewTextChild(gen_node, ns_common, BAD_CAST "schema_version", BAD_CAST generator->schema_version);
+	if (generator->core_schema_version)
+		xmlNewTextChild(gen_node, ns_common, BAD_CAST "schema_version", BAD_CAST generator->core_schema_version);
+
+	const char *namespace_uri = "http://oval.mitre.org/XMLSchema/oval-definitions-5";
+	sv_itr = oscap_htable_iterator_new(generator->platform_schema_versions);
+	while (oscap_htable_iterator_has_more(sv_itr)) {
+		oscap_htable_iterator_next_kv(sv_itr, &platform, (void **) &version);
```

Comment:
Oh, sorry. I thought you can pass <anything>* to void* by standard, But gcc gives me some warning. I have learned a new thing :]